src/QUESTIONS.md

```diff
@@ -2,8 +2,16 @@
 
 ### What is CHERI?
 
-See
-[An Introduction to CHERI](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-941.pdf).
+CHERI (Capability Hardware Enhanced RISC Instructions) is an international research project to revisit the fundamental security assumptions in both hardware and software. [An Introduction to CHERI](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-941.pdf) provides a high-level overview of the project. It builds upon an earlier fruit-themed project known as [BERI](https://www.cl.cam.ac.uk/research/security/ctsrd/beri/).
```

Comment:
AFAIK BERI was not earlier, the two were concurrently developed, with BERI's existence being specifically to enable CHERI from the start.